pkg/operator/controller/names.go

```diff
@@ -40,6 +40,8 @@ const (
 	// DefaultCanaryNamespace is the default namespace for
 	// the ingress canary check resources.
 	DefaultCanaryNamespace = "openshift-ingress-canary"
+
+	RemoteWorkerLabel = "node.openshift.io/remote-worker"
```

Comment:
Would you mind adding godoc for this definition?  

Defining the label here is fine as a temporary measure, but will it eventually be defined in openshift/api or the like?

Reply:
I hope yes, currently we want to start using it and there is no actual need to add it to openshift-api right a way. 
But yes, when we will start increasing usage, we will move it to openshift-api